layout/header.scss

```diff
@@ -0,0 +1,95 @@
+header{
+    #background-layout{
+        background-color: #fff;
+        margin-top: 20px;
+    }
+    .header-layout{
+        /* width: 100%; */
+        padding: 0 5rem;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        height: 30px;
+        background: #f4f6f7;
+        .logo-icon{
+            display: flex;
+            align-items: center;
+            .h1-text{
+                font-size: 18px;
+            }
+        }
+        .logo-icon:hover{
+            cursor: pointer;
+        }
+        .header-layout-icon{
+            display: flex;
+            position: relative;
+            .arrow-left {
+                position: relative;
+                background-color: #00bcd4;
+                margin-left: 10px;
+                width: 20px;
+                height: 10px;
+            }
+            .arrow-left::before{
+                position: absolute;
+                content: "";
+                width:0px;
+                height:0px;
+                border-bottom: 5px solid transparent;
+                border-top: 5px solid transparent;
+                border-right: 5px solid #00bcd4;
+                margin-left: -5px;
+            }
+        }
+        .btn-menu{
+            display: none;
+        }
+        .header-layout-list-link{
+            display: flex;
+        }
+        .header-layout-list-link > a {
+            margin-right: 5px;
+        }
+        .header-layout-list-link > a:hover{
+            cursor: pointer;
+            color: #00bcd4;
+        }
+    }
+    .flex{
+        // tìm cách format file css, scss
```

Comment:
tìm cách sửa cái comment với xóa comment của a đi chứ.
ai lại để .flex to, bên trong là nested thế e. ở ngoài class rõ nghĩa hơn chứ